configure.py

```diff
@@ -1117,6 +1119,25 @@ def query_seastar_flags(seastar_pc_file, link_static_cxx=False):
     modes[mode]['seastar_cflags'] = seastar_cflags
     modes[mode]['seastar_libs'] = seastar_libs
 
+MODE_TO_CMAKE_BUILD_TYPE = {'release' : 'RelWithDebInfo', 'debug' : 'Debug', 'dev' : 'Dev', 'sanitize' : 'Sanitize' }
+
+def configure_zstd(build_dir, mode):
+    zstd_build_dir = os.path.join(build_dir, mode, 'zstd')
```

Comment:
Please add a comment explaining that we use features not available in the dynamically linked version of zstd (and which features, too) so people don't try to optimize this away.

Reply:
Done